src/model/users.rs

```diff
@@ -358,9 +381,23 @@ pub struct CurrentAccount {
 	pub offline_friends: Vec<crate::id::User>,
 	/// Friends that are online
 	pub online_friends: Vec<crate::id::User>,
+	/// Friends that are active
+	pub active_friends: Vec<crate::id::User>,
 	/// Either a date-time or empty string.
 	#[serde(rename = "last_login", with = "rfc3339")]
 	pub last_login: OffsetDateTime,
+	/// When the entry was updated
+	#[serde(rename = "updated_at", with = "rfc3339")]
+	pub updated_at: OffsetDateTime,
+	/// The current avatar id or empty string.
+	#[serde(default)]
+	pub current_avatar: String,
+	/// The current avatar tags or empty string.
+	#[serde(default)]
+	pub current_avatar_tags: serde_json::Value,
+	/// The fallback avatar id or empty string.
+	#[serde(default)]
+	pub fallback_avatar: String,
```

Comment:
... not sure if this is correct, my testing account's API responses never had this...